engine/src/flutter/.ci.yaml

```diff
@@ -457,6 +457,36 @@ targets:
       # at https://github.com/flutter/flutter/issues/152186.
       cores: "8"
 
+  - name: Linux mac_clang_tidy
+    recipe: engine_v2/engine_v2
+    bringup: true
+    properties:
+      config_name: mac_clang_tidy
+    runIf:
+      - DEPS
+      - engine/src/flutter/.ci.yaml
+      - engine/src/flutter/tools/clang_tidy/**
+      - engine/src/flutter/ci/builders/**
+      - engine/src/flutter/ci/clang_tidy.sh
+      - "engine/src/flutter/**.h"
```

Comment:
These are just regex right? Can it not be:

```
      - "engine/src/flutter/**.(h|c|cc|fbs|frag|vert|m|mm)"
```

Reply:
I don't think it's regex, I think it's glob-format

Reply:
In cocoon, its regex. They change "**" to a pattern.

Reply:
Done.